### PR TITLE
Export React components

### PR DESCRIPTION
**Pull request by richiewenn**
Use Rollup to bundle js files and postcss-cli for bundling the css files.

### CODE REVIEW

File: src/client/components/ReactComponent.jsx [hunk 1]
```diff
@@ -25,12 +25,20 @@ const ReactComponent = ({ path, metadata }) => {
...
     const examples = metadata.examples || [];
     const componentName = extractComponentNameFromPath(path);
+    let markdownHtml;
+    try { // Do not fail in case of missing documentation
+        markdownHtml = marked(metadata.description)
+    } catch (e) {
+        markdownHtml = metadata.description
+        console.warn(metadata.description)
+        console.warn(e)
+    }
```
Comment (alexyork) [inline src/client/components/ReactComponent.jsx]: Everywhere else in the codebase uses semi-colons at the end of statements. You should do the same, for consistency.

File: src/components/index.js [hunk 2]
```diff
@@ -1,9 +1,10 @@
-import { getReactComponentsMetadata } from '../componentMetadata';
-
-const componentMetadata = getReactComponentsMetadata();
+// import { getReactComponentsMetadata } from '../componentMetadata';
+//
+// const componentMetadata = getReactComponentsMetadata();
```
Comment (alexyork) [inline src/components/index.js]: Remove these commented-out lines?

File: webpack-dev.config.js [hunk 3]
```diff
@@ -41,17 +41,21 @@ module.exports = {
...
+                    {loader: 'style-loader'},
+                    {loader: 'css-loader', options: {importLoaders: 1}},
+                    {loader: 'postcss-loader'}
                 ],
                 exclude: /node_modules/
             },
+            // {
+            //     test: /\.(jpe?g|png|ttf|eot|svg)(\?[a-z0-9=&.]+)?$/,
+            //     use: 'base64-inline-loader?name=[name].[ext]'
+            // },
```
Comment (alexyork) [inline webpack-dev.config.js]: Remove?

File: .flowconfig [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+[include]
+
+[libs]
+
+[lints]
+
+[options]
+
+[strict]
```
Comment (alexyork) [inline .flowconfig]: Should this file be included in the repo?

Comment (richiewenn) [reply .flowconfig]: I think, it should. It is similar case as including .eslintrc in the repository.

**Pull request merged**